MekHQ/src/mekhq/gui/model/PartsInUseTableModel.java

```diff
@@ -400,4 +409,24 @@ public Component getTableCellRendererComponent(JTable table, Object value, boole
             return renderButton;
         }
     }
+
+    @Override
+    public void setValueAt(Object value, int rowIndex, int columnIndex) {
```

Comment:
Do we want to do this, or would it be better to use a JSpinner?

Reply:
Being able to type it in fits a lot better into the table layout, and the typing + the regex makes it pretty easy to edit, unless someone really wants the change, the typeable box works very well.

Reply:
Solid reasoning :)